drivers/sensor/nordic/qdec_nrfx/qdec_nrfx.c

```diff
@@ -87,8 +103,7 @@ static int qdec_nrfx_channel_get(const struct device *dev,
 	}
 
 	key = irq_lock();
-	acc = data->acc;
-	data->acc = 0;
+	acc = data->fetched_acc;
```

Comment:
This change will break https://github.com/nrfconnect/sdk-nrf/blob/main/applications/nrf_desktop/src/hw_interface/wheel.c#L119

Fetch is not called there - probably an overlook.

Reply:
Thanks for pointing this out, wheel updated. Could not find any other occurrences beside the one you mentioned.